examples/07-Train-an-xgboost-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>up to you: I wouldn't bother including the apostrophe in the inline code style.</p><p> </p><p>Instead of "recently introduced," can you say something like "This class is introduced in the XGBoost 1.1 release and this data format provides..."</p><p> </p><p>Instead of a link with "here" as the text, can you use "You can read more about it in this [article](...) from the RAPIDS AI chann<a href="..." target="_blank">article</a></p><p> </p><p>more present tense sugg: "...with early stopping, the training ceases as soon as the model starts overfitting..."</p><p> </p><p>Maybe: "The <code>verbose_eval</code> parameter specifies..."  Up to you. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>